polus-k-means-clustering-plugin/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM labshare/polus-bfio-util:1.0.8-slim-buster
+FROM python:3.6-slim-buster
```

Comment:
In another comment, I recommend removing pandas. This should simplify the build. Since this code should only require numpy and scikit learn, please attempt to build with alpine rather than slim-buster. The numpy installation can be a little tricky, but you can use the bfio Dockerfile to see how it was done there.